homeassistant/components/improv_ble/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "improv_ble",
+  "name": "Improv via BLE",
+  "bluetooth": [{ "service_uuid": "00467768-6228-2272-4663-277478268000" }],
```

Comment:
`service_uuid`s are effectively forever since once we see it, it will always be in the advertisement data until the device goes out of range.  This means that we will see it again even after the device is provisioned

Reply:
Usually a device does something with the manufacturer or service data to update a specific uuid to change the state from unpaired to paired, or in this case un-provisioned to provisioned

Reply:
I think this can be solved by moving `00467768-6228-2272-4663-277478268000` into service data and advertising the state https://www.improv-wifi.com/ble/.  byte0 could could be the capabilities, byte1 could be the current state, byte 2/3 reserved for future.  When the state changes it would update the adv.  Than the config flow would be able to ignore devices that didn't have `0x01    Authorization Required`    Awaiting authorization via physical interaction. in byte1